app/com/lucidchart/open/cashy/controllers/BrowseController.scala

```diff
@@ -33,19 +33,25 @@ class BrowseController extends AppController {
 
       val listResponse = S3Client.listObjects(bucket, browsePath, marker)
 
+      val foldersHidden = FolderModel.findByKeys(bucket, listResponse.folders).map(folder => (folder.key, folder)).toMap
```

Comment:
It might be more accurate to rename this variable and the `assetsHidden` variable to something that doesn't include the word "hidden", as what the models are returning are all assets and folders, not the ones that are hidden.